chain/src/store.rs

```diff
@@ -140,6 +140,13 @@ impl ChainStore {
 		}
 	}
 
+	pub fn get_hash_by_height(&self, height: u64) -> Result<Hash, Error> {
```

Comment:
Nice. Saves us a redundant read from the db each time (and reading the full block header just to get the hash).

Reply:
I think this has almost been added several times in the past...